Com.OneSignal.Android/OneSignalImplementation.cs

```diff
@@ -165,7 +165,12 @@ public override void SetExternalUserId(string externalId, OnExternalUserIdUpdate
       {
          Android.OneSignal.SetExternalUserId(externalId, new OSExternalUserIdUpdateCompletionHandler(completion));
       }
-      
+
+      public override void SetExternalUserId(string externalId, string authHashToken, OnExternalUserIdUpdate success, OnExternalUserIdUpdateFailure failure)
+      {
+         Android.OneSignal.SetExternalUserId(externalId, authHashToken, new OSExternalUserIdUpdateCompletionHandler(success));
```

Comment:
On iOS we handle `failure` however here in Android it is just ignored.

Reply:
yeah because on android we only have a completion, in the major release the failure block is added. This was to avoid breaking changes on master

Reply:
ah gotcha, I see Android native does not have a failure handler. Sorry forgot about that.